src/wallet-user/transaction-protection/LocCreation.tsx

```diff
@@ -50,6 +50,7 @@ export default function LocCreation(props: Props) {
                 legalOfficer: selectedLegalOfficer!,
                 description: formValues.description,
                 locType,
+                draft: false,
```

Comment:
I assume the actual draft mode will be part of the scope of an upcoming PR.

Reply:
Yes indeed